functions/functions/messages/list-messages.js

```diff
@@ -0,0 +1,12 @@
+const { listMessages } = require(Runtime.getFunctions()["twilio/messages"].path);
```

Comment:
Can we discuss via slack the pros / cons of requiring the function instead of just rewriting the api function itself as the function? I'm kind of torn here.